### DISCUSSION TITLE
Is `/` inside the slug supported?

### DISCUSSION THREAD

**Discussion by xmontero** [shlinkio/discussions#89] [Help wanted] [answered]
#### How Shlink is set-up

* Shlink Version: Server: [v3.3.2] - Client: [v3.7.3]
* How do you serve Shlink: Docker image
* Database engine used: MySQL

#### Summary

First, congrats for this great project, Alejandro.

Second: A URL redirector is not always to shorten URLs but sometimes for branding them.

I have in place some scheme like those:

* https://go.my-nice-brand.example.com/challenge/sing-a-song
* https://go.my-nice-brand.example.com/challenge/paint-a-painting
* https://go.my-nice-brand.example.com/invite/this-affiliate-personal-name
* https://go.my-nice-brand.example.com/invite/the-other-affiliate-personal-name

I find that whenever I put a `/` in the slug it gets converted into `-`, probably to match the "router" pattern /{slug}

I wonder if via configuration the router pattern can be configured to accept a `/` in the middle so it'll catch `/rest` first and then "anything else" like `/abc` or `/abc-xyz` or even `/abc/xyz` to enter into the "slug".

If not via config, I'm happy to try to improve the code if you give the tips to start with.

**Comment by acelaya** [comment, marked as answer]
This is already supported. It's called "multi-segment slugs", but you need to actively opt in. https://shlink.io/documentation/some-features/#multi-segment-custom-slugs

**Reply by xmontero** [reply]
Tested! That's what I needed. Thanks!

**Reply by xmontero** [reply]
xref https://github.com/shlinkio/shlink.io/pull/325 - I made a PR to the docs.
